search/task_helpers.py

```diff
@@ -349,3 +352,44 @@ def update_indexed_score(instance, instance_type, vote_action=None):
         incr_amount=vote_increment,
         object_type=instance_type,
     )
+
+
+@if_feature_enabled(INDEX_UPDATES)
+def index_new_course(course_obj):
```

Comment:
This doesn't seem to be used anywhere aside from tests

Reply:
Usage must have been inadvertently removed during a previous rebase, added back (ditto for update_course)